src/main/resources/assets/admin/common/styles/api/form/set/form-set-view.less

```diff
@@ -1,207 +1,216 @@
-    border: 2px dashed #000;
-    padding: 14px;
-    color: #000;
-    font-size: 14px;
-    text-align: center;
-    margin: 10px;
-  }
-
-  .input-view {
-    .input-label {
-      max-width: 90px;
-      margin-right: 10px;
+    .form-item-set-drop-target-placeholder, .form-option-set-drop-target-placeholder {
+      border: 2px dashed #000;
+      padding: 14px;
+      color: #000;
+      font-size: 14px;
+      text-align: center;
+      margin: 10px;
     }
 
-    .input-type-view,
-    .bottom-button-row,
-    .validation-viewer {
-      margin-left: @form-label-width;
-    }
+    .input-view {
+
+      &.invalid {
+        padding: 10px;
+
+        .validation-viewer ul > li {
+          margin-top: 10px;
+        }
+      }
 
-    &.label-over-input {
       .input-label {
-        max-width: 100%;
+        max-width: 90px;
+        margin-right: 10px;
       }
+
       .input-type-view,
       .bottom-button-row,
       .validation-viewer {
-        margin-left: 0;
+        margin-left: @form-label-width;
       }
-    }
-  }
-
-  & > .bottom-button-row {
-    box-sizing: border-box;
-    background-color: @admin-bg-light-gray;
-    padding: 10px;
-    clear: left;
-    .clearfix();
-
-    button {
-      margin-left: 0;
-    }
 
-    .add-button {
-      float: left;
-    }
-
-    .collapse-button {
-      float: right;
-      text-decoration: underline;
-      line-height: 28px;
-      margin-right: 10px;
+      &.label-over-input {
+        .input-label {
+          max-width: 100%;
+        }
 
-      &:hover {
-        cursor: pointer;
+        .input-type-view,
+        .bottom-button-row,
+        .validation-viewer {
+          margin-left: 0;
+        }
       }
     }
-  }
 
-  & > .validation-viewer {
-    background-color: @admin-bg-light-gray;
-    padding: 0 10px 10px 10px;
-    margin-left: 0 !important;
-  }
+    & > .bottom-button-row {
+      box-sizing: border-box;
+      padding: 5px 0 0;
+      clear: left;
+      .clearfix();
 
-  &.max-1-occurrence {
-    margin-bottom: 0;
+      button {
+        margin-top: 5px;
+      }
 
-    .occurrence-views-container {
-      > .form-option-set-occurrence-view {
-        margin-bottom: 0;
-        padding: 0;
+      .add-button {
+        float: left;
       }
 
-      &:empty + .bottom-button-row {
-        display: block;
-        padding-left: 0;
-        padding-bottom: 0;
-        background-color: @admin-white;
+      .collapse-button {
+        float: right;
+        text-decoration: underline;
+        line-height: 28px;
+
+        &:hover {
+          cursor: pointer;
+        }
       }
     }
 
-    > .bottom-button-row {
-      display: none;
+    & > .validation-viewer {
+      background-color: @admin-bg-light-gray;
+      padding: 0 10px 10px 10px;
+      margin-left: 0 !important;
     }
 
-  }
+    &.max-1-occurrence {
+      .occurrence-views-container {
+        > .form-option-set-occurrence-view {
```

Comment:
Codacy found an issue: [Expected selector ".form-view .form-item-set-view.max-1-occurrence .occurrence-views-container > .form-option-set-occurrence-view" to come before selector ".form-view .form-item-set-view.multiple-occurrence > .occurrence-views-container .form-option-set-occurrence-view:not(:last-child)" (no-descending-specificity)](https://app.codacy.com/manual/enonic/lib-admin-ui/pullRequest?prid=6371096)